end_users_section.html

```diff
@@ -0,0 +1,17 @@
+<div class="section-title" style="text-align: center; margin-top: 120px; margin-bottom: 60px; display: flex; justify-content: center; align-items: center;">
+    <h2>End Users</h2>
+  </div>
+  
+  <div class="logos-container" style="display: flex; flex-direction: row; justify-content: center; align-items: center; margin-bottom: 30px;">
```

Comment:
Since the styles are defined inline seems to me like you can remove the class attributes

Reply:
fixed